locale/ru_RU/countries.xml

```diff
@@ -35,7 +32,7 @@
 	<country name="Бахрейн" code="BH"/>
 	<country name="Бангладеш" code="BD"/>
 	<country name="Барбадос" code="BB"/>
-	<country name="Беларусь" code="BY"/>
+	<country name="Белоруссия" code="BY"/>
```

Comment:
Does this correspond to the Debian ISO 3166-1 iso-codes package, which we're generally using for translations of this file? I'm nervous about inadvert politics. (See the first paragraph of https://en.wikipedia.org/wiki/Belarus)